packages/account/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@biconomy/bundler-rpc",
+  "name": "@biconomy/account",
   "version": "2.0.0",
-  "description": "Biconomy Bundler Rpc to interact with any bundler",
+  "description": "Biconomy account enable gasless transaction for any smart wallet",
```

Comment:
grammar / description is bit off. 
also use the word account or contract account instead of wallet please.

Reply:
what this package is really doing is providing Account APIs for different contract account implementations.